commit c29dbc5c54b6e7daa472ba0a0d04666dc49f7523
Author: Oleg Drokin <green@linuxhacker.ru>
Date:   Sun Feb 19 16:35:59 2017 -0500

    staging/lustre/lnet: Fix allocation size for sv_cpt_data
    
    commit dc7ffefdcc28a45214aa707fdc3df6a5e611ba09 upstream.
    
    This is unbreaking another of those "stealth" janitor
    patches that got in and subtly broke some things.
    
    sv_cpt_data is a pointer to pointer, so need to
    dereference it twice to allocate the correct structure size.
    
    Fixes: 9899cb68c6c2 ("Staging: lustre: rpc: Use sizeof type *pointer instead of sizeof type.")
    CC: Sandhya Bankar <bankarsandhya512@gmail.com>
    Signed-off-by: Oleg Drokin <green@linuxhacker.ru>
    Reviewed-by: James Simmons <jsimmons@infradead.org>
    Reviewed-by: Doug Oucharek <doug.s.oucharek@intel.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1242_c29dbc5c54b6e7daa472ba0a0d04666dc49f7523.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1242_c29dbc5c54b6e7daa472ba0a0d04666dc49f7523.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
